ci-operator/config/kube-reporting/metering-operator/kube-reporting-metering-operator-release-4.6.yaml

```diff
@@ -48,6 +48,22 @@ images:
 - dockerfile_path: olm_deploy/Dockerfile.registry
   from: base
   to: metering-ansible-operator-registry
+operator:
+  bundles:
+  - dockerfile_path: Dockerfile.bundle
```

Comment:
This Dockerfile already exists in the repo: https://github.com/kube-reporting/metering-operator/blob/master/Dockerfile.bundle